deb/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
```

Comment:
This is an extra effort but this script could be made portable by using the command line command of the cmake binary. See here: https://cmake.org/cmake/help/latest/manual/cmake.1.html#run-a-command-line-tool